### PR TITLE
Fixes to Vagrantfile

### PR DESCRIPTION
**Pull request by sciabarracom**
The Vagrantfile does not work in Windows because of the hardcoded slashed. I fixed this.
Also the wsk property configuration in the Vagrantfile does not reflect the current setup, so I changes to use http: and 10001.

### CODE REVIEW

**Comment by rabbah** [comment]
Thanks for the fixes!

File: tools/vagrant/Vagrantfile [hunk 1]
```diff
@@ -159,7 +159,7 @@ $script_end = <<SCRIPT
   touch $HOME/.wskprops
   chown -R vagrant:vagrant $HOME
   # This allows user to see how to configure the wsk cli outside the VM
-  wsk property set --apihost ${WHISK_IP} --namespace guest --auth `cat ${ANSIBLE_HOME}/files/auth.guest`
+  wsk property set --apihost http://${WHISK_IP}:10001 --namespace guest --auth `cat ${ANSIBLE_HOME}/files/auth.guest`
```
[the remark this answers is not in the document]
Comment (csantanapr) [reply tools/vagrant/Vagrantfile]: wait what? WTF
That explains why you tried to change it.

I will give it a try now on Mac. I don't have a Windows machine

Can you help debug:
Can you run `docker ps` and paste that here
That should tell us the containers running and the ports exposed, and we should see `443` exposed from nginx

For example on native mac:
```
😄  $ docker ps | grep nginx
86f32aea8538        nginx:1.11                    "nginx -g 'daemon ..."   13 hours ago        Up 13 hours         0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 0.0.0.0:8443->8443/tcp         nginx
~
😄  $ curl -k https://0.0.0.0:443/api/v1
😄  $ curl -k https://localhost:443/api/v1
😄  $ curl -k https://172.17.0.1:443/api/v1
{"api_version_path":"v1","description":"OpenWhisk API","swagger_paths":{"ui":"/docs","api-docs":"/api-docs"},"build":"2018-01-12T05:09:21Z","api_version":"1.0.0","buildno":"latest"}
```

[the remark this answers is not in the document]
Comment (sciabarracom) [reply tools/vagrant/Vagrantfile]: First note the error is (inside the vm, AFTER a vagrant halt / vagrant up - I know docs says to use susped)  :

$ wsk action list
error: Unable to obtain the list of actions for namespace 'guest': Get https://192.168.33.13/api/v1/namespaces/guest/actions?limit=30&skip=0: dial tcp 192.168.33.13:443: getsockopt: connection refused

Then:
$ docker ps | grep nginx
aeb333ed2b95        nginx:1.11                    "nginx -g 'daemon off"   5 hours ago         Restarting (1) About an hour ago   0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 0.0.0.0:8443->8443/tcp         nginx
$ curl -k https://0.0.0.0:443/api/v1
curl: (35) Unknown SSL protocol error in connection to 0.0.0.0:443
$ curl -k https://localhost:443/api/v1
curl: (35) Unknown SSL protocol error in connection to localhost:443
$ curl -k https://172.17.0.1:443/api/v1
curl: (7) Failed to connect to 172.17.0.1 port 443: Connection refused

I am pretty sure the reason of this is the halt/up but I was trying to understand why a restart of a vm is so destructive.

[the remark this answers is not in the document]
Comment (rabbah) [reply tools/vagrant/Vagrantfile]: https://github.com/apache/incubator-openwhisk/issues/32 may be at play.

**Comment by sciabarracom** [comment]
Ok just the fix to the vagrantfile for Windows.

**Comment by mrutkows** [comment]
Thanks for getting Windows Vagrant healthy!

**Pull request closed without merging**